juriscraper/opinions/united_states/state/ark.py

```diff
@@ -2,60 +2,103 @@
-            name = titlecase(name.strip(" -"))
-            cite = cite.strip(" -")
-            date = date.strip(" -")
-
-            if int(date[-4:]) != self.year:
-                # The RSS feed has the most recent opinions and then dozens from the 1960's.  Not sure why
-                # but we dont want to collect those.
-                continue
-            if len(self.cases) >= 10:
-                # Limit to max of 10 cases per crawl to stay under limit.
-                # We assume it is the same for NM (lexum) at 50/min 100/5min
+        """Parse HTML into case dictionaries
+
+        XMLHttpRequest pagination is triggered every 25 rows, so we must
+        try to avoid big date intervals
+
+        :return None
+        """
+        rows = self.html.xpath("//div[@class='info']")
+        if len(rows) >= 25:
+            logger.info(
+                "25 results for this query, results may be lost in pagination"
+            )
+
+        for item in rows:
+            per_curiam = False
+
+            name = item.xpath(".//a/text()")[0]
+            url = item.xpath(".//a/@href")[1]
+            if re.search(self.not_a_opinion_regex, name.upper()):
+                logger.info("Skipping %s %s, invalid document", name, url)
                 continue
-            doc_id = re.findall(r"\d{4,}", item["link"])[0]
-            url = f"https://opinions.arcourts.gov/ark/{self.court}/en/{doc_id}/1/document.do"
+
+            cite = item.xpath(".//*[@class='citation']/text()")
+            if cite:
+                cite = cite[0]
+
+            if metadata := item.xpath(".//*[@class='subMetadata']/span[2]"):
+                per_curiam = metadata[0].text_content().strip() == "Per Curiam"
+
+            date_filed = item.xpath(".//*[@class='publicationDate']/text()")[0]
             self.cases.append(
                 {
-                    "date": date,
+                    "date": date_filed,
                     "docket": "",
                     "name": name,
                     "citation": cite,
                     "url": url,
                     "status": "Published",
+                    "per_curiam": per_curiam,
                 }
             )
 
+    def set_url(
+        self, start: Optional[date] = None, end: Optional[date] = None
+    ) -> None:
+        """Formats and sets `self.url` with date inputs
+
+        If no start or end dates are given, scrape last 7 days.
+
+        :param start: start date
+        :param end: end date
+
+        :return None
+        """
+        if not start:
+            end = datetime.now() + timedelta(1)
+            start = end - timedelta(7)
+
+        params = {
+            "cont": "",
+            "ref": "",
+            "d1": start.strftime("%m/%d/%Y"),
+            "d2": end.strftime("%m/%d/%Y"),
+            "col": self.court_code,
+            "tf1": "",
+            "tf2": "",
+            "or": "date",
+            "iframe": "true",
```

Comment:
do we need all of these empty fields or can we remove them.

Reply:
I think without them it stops working

Just commented them out and the run `arkctapp` normal scraper 
- Without: 0 results
- With: 1 result (Skipping 07-17-24 SYLLABUS)